### DISCUSSION TITLE
Start Engine button issue for the A320

### DISCUSSION THREAD

**Discussion by KennethStorm** [nguyenquyhy/Flight-Tracker-StreamDeck#193] [Q&A] [answered]
Hi
Nguyenquyhy Great work :)

I have made 2 Generic Toggle Button's for the engine start on the A320 with the 'TOGGLE_FUEL_VALVE_ENG1' and 'TOGGLE_FUEL_VALVE_ENG2' but can't find the feedback value for that, so i can se on the stream deck that it is on, and that the images changes.
I know i can hear the engine spin up, but it could be nice to se it as on, on the stream deck!

I have tried the 'GENERAL_ENG_FUEL_VALVE__1' but that dont work :(

**Comment by nguyenquyhy** [comment, marked as answer]
Which A320 is that (default or FBW 32NX Stable/Dev/Experimental)? 

Logically the feedback should be `GENERAL ENG FUEL VALVE:1`, but aircraft dev tends to not stick to SimConnect variables 100%, limiting the interoperability with external plugins. 

I can see that GENERAL_ENG_FUEL_VALVE__1 is already supported by the plugin, so it's not really something we can do. You'll have to ask the aircraft dev to support the variable.

**Reply by KennethStorm** [reply]
Hi

It is 'Dev' version, the 'Stable' version has some other issues, but ok then it is right what i have done :)
just not supported in the 'Dev' version.
Thanx for guick responce :)

Best Regards
Kenneth Storm
